### ISSUE TITLE
[BUG]: Mobile Navbar is Unscrollable in Landscape Orientation (WCAG, RWD)

### ISSUE THREAD

**Issue by jschhie** [Crustaly/audemywebsite#284]
### Describe the bug

- Main navbar in the `<Header/>` overflows the viewport when viewed in mobile landscape mode (ex: holding phone sideways)
- Navbar is cut off & users cannot scroll vertically to access the remaining links 
- Hidden/Inaccessible Links: 
  - "Games Toolkits",  "Access. Studio", & "World's 1st Audio Console" pages
  - All social media links in the mobile navbar's footer
-  RWD failure blocks user interaction & violates WCAG, which requires easy keyboard and touch access to all navigation UI 

### Steps to reproduce

1. Visit any page in on mobile
2. Rotate the device to landscape mode
3. Open navbar icon (hamburger menu)
4. Try to scroll, view, or access all navbar links
5. Notice bottom page links & socials are cut off

### Expected behavior

- Navbar links should be fully accessible via both keyboard and touch to comply w/ WCAG

### Actual behavior

- When mobile device is in landscape mode, the bottom part of the navbar is cut off and unscrollable
- **Idea Fix:** 
  - Either adjust RWD, specifically for landscape mode on mobile, to prevent overflow
  - OR implement vertical scrolling within the navbar to allow users to access all links

### Screenshots or recordings

| RWD / WCAG Bug: Inaccessible Navbar | 
| :---: |
| <img width="2778" height="1284" alt="Image" src="https://github.com/user-attachments/assets/d89a0bb9-fdcc-4f6c-b204-241ec21ec21c" /> |

### Environment details

- **Backend:** `main` @ commit [d862c12](https://github.com/Crustaly/audemywebsite/commit/d862c126881f3e60bc36aaff03aabf34b36a624e) 
- **Device:** Mobile in landscape mode

### Before submitting

- [x] I have searched for existing bug reports.
- [x] I am using the latest version of Audemy.
- [x] I can reliably reproduce this bug.

**Comment by jschhie** [comment]
WCAG fix was addressed in [PR #291](https://github.com/Crustaly/audemywebsite/pull/291) (currently, staged for `main`). 
Closing this Issue now :)
